src/System.Management.Automation/engine/SessionStateVariableAPIs.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
```

Comment:
@iSazonov, your last commit had 1166 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Enable-ExperimentalFeature will output warning message
```powershell
Cannot bind argument to parameter 'Path' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\EnableDisable-ExperimentalFeature.Tests.ps1: line 47
47:         Remove-Item $systemConfigPath -Force -ErrorAction SilentlyContinue

```
Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Disable-Experimentalfeature will output warning message
```powershell
Cannot bind argument to parameter 'Path' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\EnableDisable-ExperimentalFeature.Tests.ps1: line 47
47:         Remove-Item $systemConfigPath -Force -ErrorAction SilentlyContinue

```
Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Disable-ExperimentalFeature will disable Experimental Feature for scope: AllUsers
```powershell
Cannot bind argument to parameter 'Path' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\EnableDisable-ExperimentalFeature.Tests.ps1: line 47
47:         Remove-Item $systemConfigPath -Force -ErrorAction SilentlyContinue

```
Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Disable-ExperimentalFeature will disable Experimental Feature for scope: CurrentUser
```powershell
Cannot bind argument to parameter 'Path' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\EnableDisable-ExperimentalFeature.Tests.ps1: line 47
47:         Remove-Item $systemConfigPath -Force -ErrorAction SilentlyContinue

```
Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Enable-ExperimentalFeature will enable Experimental Feature for scope: AllUsers
```powershell
Cannot bind argument to parameter 'Path' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\EnableDisable-ExperimentalFeature.Tests.ps1: line 47
47:         Remove-Item $systemConfigPath -Force -ErrorAction SilentlyContinue

```